src/Logging/Loggly/Loggly.csproj

```diff
@@ -12,9 +12,17 @@
     <PackageTags>dotnet-standard;logging;loggly</PackageTags>
   </PropertyGroup>
 
-  <ItemGroup>
-      <PackageReference Include="Microsoft.Extensions.Logging" Version="2.1.1" />
-      <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
-      <PackageReference Include="System.Reactive.Linq" Version="3.1.1" />
+  <ItemGroup Condition=" '$(TargetFramework)' != 'net8.0' ">
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="2.1.1" />
+    <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
+    <PackageReference Include="System.Reactive.Linq" Version="4.4.1" />
   </ItemGroup>
-</Project>
\ No newline at end of file
+
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net8.0' ">
+    <PackageReference Include="Microsoft.Extensions.Configuration" Version="8.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Binder" Version="8.0.1" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="8.0.0" />
+    <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
+    <PackageReference Include="System.Reactive.Linq" Version="4.4.1" />
+  </ItemGroup>
```

Comment:
Since it didn't look like you needed to update Newtonsoft to a later version, could we just upgrade the other packages for the existing target frameworks?

Reply:
I just tested it on my local and the Newtonsoft update is something needed. Version updated.